boards/hail/src/main.rs

```diff
@@ -35,15 +35,15 @@ unsafe fn load_processes() -> &'static mut [Option<kernel::process::Process<'sta
         static _sapps: u8;
     }
 
-    const NUM_PROCS: usize = 2;
+    const NUM_PROCS: usize = 4;
```

Comment:
We need a way to statically size the `PROCESSES` array... I guess this could be `MAX_PROCS` instead.